### ISSUE TITLE
docs and source disagree on babel.core.get_global('win_mapping')

### ISSUE THREAD

**Issue by ecederstrand** [python-babel/babel#463]
According to the docs (http://babel.pocoo.org/en/latest/api/core.html#utility-functions), I should do this to get the Windows timezone mapping:

```python
babel.core.get_global('win_mapping')
```

but this collection is empty, and the actual mapping (using Babel 2.3.4 pip-installed on Ubuntu 14.04) is located at:

```python
babel.core.get_global('windows_zone_mapping')
```

So, it seems either the source code or the documentation should be updated. See also #464

**Comment by abhishekcs10** [comment]
I am new to open source. What needs to be done? Should I fix the documentation?

**Comment by akx** [comment]
Closed via #466. Thanks (and sorry for the delay).
